cluster/plugin/aws/cmd/main.go

```diff
@@ -97,8 +96,8 @@ func delete(cmd *cobra.Command, args []string) {
 		if err := svc.WaitUntilStackDeleteCompleteWithContext(ctx, input); err != nil {
 			log.Fatal(err)
 		}
-		log.Printf("stack deleted: %s\n", opts.StackName)
 	}
+	log.Printf("stack deleted: %s\n", opts.StackName)
```

Comment:
I get this:
```
amp cluster rm --provider aws --aws-region us-west-2 --aws-stackname nde-test
[localhost:50101]
/08/24 20:59:16 stack deleted: nde-test
Error: json: cannot unmarshal number into Go value of type aws.StackOutputList%
```

And if I provide a non existent stack name, I get the same answer.